docs/tutorial.rst

```diff
@@ -49,41 +49,37 @@ Step 1: Create a Client
 In order to complete an OAuth2 flow to get tokens, you must have a client or
 "app" definition registered with Globus.
 
-Navigate to the `Developer Site <https://developers.globus.org>`_ and select
-"Register your app with Globus."
-You will be prompted to login -- do so with the account you wish to use as your
-app's administrator.
+The following steps will walk you through creating a Native App to be used as your
+Globus interaction client.
 
-When prompted, create a Project. A Project is a collection of clients with a
-shared list of administrators.
-Projects let you share the administrative burden of a collection of apps.
+1. Navigate to the `Developer Site <https://app.globus.org/settings/developers>`_
 
-In the "Add" menu for your Project, select "Add new app". To follow this
-tutorial, we will specify several values you should use.
+2. Select "Register a thick client or script that will be installed and run by users on
+   their devices."
 
-Enter the following pieces of information:
+3. Select "none of the above - create a new project" and click continue.
```

Comment:
I just tested to confirm and the webapp skips this step if you have an account with no projects.
Not sure how exactly, but we should adjust. Maybe we make this a bullet point under (2)?

Reply:
Hmm, yeah i can revise

Reply:
I restructured bullet points 3 & 4 into a share one.